Dependencies/Repository/Sources/Repository/Repository.swift

```diff
@@ -23,70 +25,72 @@ public protocol TimeLogRepository {
             database.view.workspaces.changes,
             workspacesLoadSubject.map { DatabaseChanges(insertions: $0) }
         )
+        .map(DatabaseChanges.map(Workspace.fromDAO))
     }
 
-    public var clientsLoadSubject = PublishSubject<[Client]>()
+    public var clientsLoadSubject = PublishSubject<[ClientDAO]>()
     public var clientsChanges: Observable<DatabaseChanges<Client>> {
         Observable.merge(
             database.view.clients.changes,
             clientsLoadSubject.map { DatabaseChanges(insertions: $0) }
         )
+        .map(DatabaseChanges.map(Client.fromDAO))
     }
 
-    public var projectsLoadSubject = PublishSubject<[Project]>()
+    public var projectsLoadSubject = PublishSubject<[ProjectDAO]>()
     public var projectsChanges: Observable<DatabaseChanges<Project>> {
         Observable.merge(
             database.view.projects.changes,
             projectsLoadSubject.map { DatabaseChanges(insertions: $0) }
         )
+        .map(DatabaseChanges.map(Project.fromDAO))
     }
 
-    public var tasksLoadSubject = PublishSubject<[Task]>()
+    public var tasksLoadSubject = PublishSubject<[TaskDAO]>()
     public var tasksChanges: Observable<DatabaseChanges<Task>> {
         Observable.merge(
             database.view.tasks.changes,
             tasksLoadSubject.map { DatabaseChanges(insertions: $0) }
         )
+        .map(DatabaseChanges.map(Task.fromDAO))
     }
 
-    public var tagsLoadSubject = PublishSubject<[Tag]>()
+    public var tagsLoadSubject = PublishSubject<[TagDAO]>()
     public var tagsChanges: Observable<DatabaseChanges<Tag>> {
         Observable.merge(
             database.view.tags.changes,
             tagsLoadSubject.map { DatabaseChanges(insertions: $0) }
         )
+        .map(DatabaseChanges.map(Tag.fromDAO))
     }
 
-    public var timeEntriesLoadSubject = PublishSubject<[TimeEntry]>()
+    public var timeEntriesLoadSubject = PublishSubject<[TimeEntryDAO]>()
     public var timeEntriesChanges: Observable<DatabaseChanges<TimeEntry>> {
         Observable.merge(
             database.view.timeEntries.changes,
             timeEntriesLoadSubject.map { DatabaseChanges(insertions: $0) }
         )
+        .map(DatabaseChanges.map(TimeEntry.fromDAO))
     }
   
-    internal init(api: TimelineAPI, database: Database, userDefaults: UserDefaultsService, time: Time) {
-        self.api = api
+    internal init(database: Database, userDefaults: UserDefaultsService, time: Time) {
         self.time = time
         self.userDefaults = userDefaults
         self.database = database
     }
 
-    public static let live = Repository(api: API.live, database: Database(), userDefaults: UserDefaultsService(), time: Time())
-
-    public static let local = LocalDataRepository(api: API.local, database: Database(), userDefaults: UserDefaultsService(), time: Time())
+    public static let live = Repository(database: Database(), userDefaults: UserDefaultsService(), time: Time())
```

Comment:
Does it still make sense to call this singleton `live`? 
```suggestion
    public static let instance = Repository(database: Database(), userDefaults: UserDefaultsService(), time: Time())
```

Reply:
My guess is we'll end up having mock ones. Let's leave it for now.